components/CheckoutActions/CheckoutActions.js

```diff
@@ -13,22 +12,24 @@ import { addTypographyStyles } from "@reactioncommerce/components/utils";
 import withAddressValidation from "containers/address/withAddressValidation";
 import Dialog from "@material-ui/core/Dialog";
 import PageLoading from "components/PageLoading";
-import { Router } from "routes";
-import track from "lib/tracking/track";
-import TRACKING from "lib/tracking/constants";
-import trackCheckout from "lib/tracking/trackCheckout";
-import trackOrder from "lib/tracking/trackOrder";
-import trackCheckoutStep from "lib/tracking/trackCheckoutStep";
+import Router from "translations/i18nRouter";
+// import track from "lib/tracking/track";
+// import TRACKING from "lib/tracking/constants";
+// import trackCheckout from "lib/tracking/trackCheckout";
+// import trackOrder from "lib/tracking/trackOrder";
+// import trackCheckoutStep from "lib/tracking/trackCheckoutStep";
```

Comment:
Ah, yes, I wanted to discuss this, but didn't investigate options yet, I agree these should definitely be adressed prior merging.
It became clear that the current approach with the tracking decorators wouldn't really be suitable for further development, and also was not really comfortable to work with.

During removal of the custom babel configuration I had not decided yet what to do with them, but did not want to remove all the references to tracking functionality upfront, in case there might be a simple way to adapt them.

Im open for suggestions here - I could just remove them from now, as we could also lookup references from the existing branches - Also, lot's of them seem obsolete and could probably better be abstracted in the hook for the respective action instead.

In the long run, I would need to research how to use react-tracking with hooks now, and maybe also evauate if it still is a good choice.
Would tracking be a priority for an initial merge?

Reply:
Removed in https://github.com/reactioncommerce/example-storefront/pull/667/commits/f99f9adf797a20148632641ef9413b915e711c56